enhancements/business-os/identity-and-access-management/authentication/README.md

```diff
@@ -399,6 +399,134 @@ discover the trusted JWKS that are use to validate that JWT authentication
 tokens provided by clients are considered valid. All services should be updated
 to trust JWT tokens signed by the new authentication system.
 
+### Login Flow For A Registrated User
+
+This sequence flow illustrates the authentication process using the built-in Zitadel UI when a user logs in via an external identity provider (e.g. GitHub, Google). 
```

Comment:
I would remove “Zitadel” from here since this should really be implementation-agnostic. It doesn’t really matter which auth provider we use; we should be able to maintain the same login flow.